src/zope/app/tree/tests/basetest.py

```diff
@@ -14,17 +14,20 @@
 """Base Test Case for Tree Tests
 
 """
+from zope.publisher.interfaces.browser import IDefaultBrowserLayer
```

Comment:
the order looks wrong here

Reply:
Only the `meta/config` template named `zope-package` uses isort.
I personally do not care much about import sort orders. (I personally sort them just alphabetically without using any groups – as this seems to be the easiest sorting.) IMHO there are more important things.